dashboard/src/components/ChartView/ChartReadme.tsx

```diff
@@ -13,6 +13,23 @@ interface IChartReadmeProps {
   version: string;
 }
 
+// Code from https://github.com/rexxars/react-markdown/issues/69
+function flatten(text: string, child: any): any {
```

Comment:
If we keep this function in this class, we should probably name it better, or instead just have it as an anonymous function in the `.reduce`

Reply:
I moved it to a different component file. I prefer to leave it as it is to maintain fidelity to the linked issue.